yt/frontends/cf_radial/data_structures.py

```diff
@@ -278,10 +278,10 @@ def _parse_parameter_file(self):
             self.domain_left_edge = np.array([x.min(), y.min(), z.min()])
             self.domain_right_edge = np.array([x.max(), y.max(), z.max()])
             self.dimensionality = 3
-            dims = [xr_ds_handle.dims[d] for d in "xyz"]
+            dims = [xr_ds_handle.sizes[d] for d in "xyz"]
             self.domain_dimensions = np.array(dims, dtype="int64")
             self._periodicity = (False, False, False)
-            self.current_time = float(xr_ds_handle.time.values)
+            self.current_time = float(xr_ds_handle.time.values[0])
```

Comment:
this is a fix for the np deprecation warning:

> DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)

Selecting the first index in the time array is consistent with choosing the first index of origin_latitude and origin_longitude (up around lines 275,276), which are both arrays that have time as a dimension.

Reply:
If we're going to convert to a Python `float` anyway, I believe this is the idomatic way to do it
```suggestion
            self.current_time = xr_ds_handle.time.values.item()
```

Reply:
So I ended up switching this back because the `time` coordinate here will typically have a datetime  dtype, and `.item()` will cast it to int. But I think it's more appropriate to cast to float here (both cause `current_time` will end up being cast to float when it's upgrade to a unyt quantity and also cause internally, xarray will encode/decode datetime objects with floats, so it's helpful to be consistent with that).